src/main/java/racingcar/dao/CarDao.java

```diff
@@ -0,0 +1,13 @@
+package racingcar.dao;
+
+import racingcar.entity.CarEntity;
+
+import java.util.List;
+
+public interface CarDao {
+
+    void insert(List<CarEntity> carEntities, long gameId);
```

Comment:
```suggestion
    void insert(List<Car> cars, long gameId);
```
Cars를 받아도 될 것 같아서요~!

Reply:
이번에 `Layered architecture`를 공부하면서 든 생각인데요,
`Presentation layer` - `Business layer` - `Persistence Layer` - `Database Layer` 이렇게 있으면,
`Dao`는 `Persistence Layer`에 해당하게 되는데, `Cars`로 받게되면 `Persistence layer`에서 `business layer`의 객체를 가져다가 쓰는 것인것 같아서요! 계층형 아키텍처에서 상위계층은 하위계층에 의존할 수 있어도, 하위 계층은 상위계층에 의존할수 없다고 공부해서, 하위계층에서 상위계층의 `Cars`를 사용하는 것은 어색할 것 같다고 생각했습니다!